independent-publisher-connectors/PaySpace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4476 @@
+                      "OldEmployeeId": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "OldEmployeeId"
+                      },
+                      "CustomFields": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "Code": {
+                              "type": "string",
+                              "description": "Code"
+                            },
+                            "Label": {
+                              "type": "string",
+                              "description": "Label"
+                            },
+                            "Value": {
+                              "type": "string",
+                              "description": "Value"
+                            }
+                          }
+                        },
+                        "description": "CustomFields"
+                      }
+                    }
+                  },
+                  "description": "value"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/odata/v1.1/{company-id}/EmployeePosition": {
+      "get": {
+        "summary": "Get a collection of positions",
+        "description": "Retrieves a list of employee position records as of the today's date and based on the specified company_id.",
+        "operationId": "GetACollectionOfPositions",
+        "parameters": [
+          {
+            "name": "$top",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": " (integer($int32)) - Limits the number of items returned from a collection",
+            "x-ms-summary": "Optional $top"
+          },
+          {
+            "name": "$skip",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": " (integer($int32)) - Excludes the specified number of items of the queried collection from the result",
+            "x-ms-summary": "Optional $skip"
+          },
+          {
+            "name": "$count",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": " (true/false) - Return total count in response",
+            "x-ms-summary": "Optional $count"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": " Filter by field eg. \"EmployeeNumber eq 'Emp01'\"",
+            "x-ms-summary": "Optional $filter"
+          },
+          {
+            "name": "$expand",
+            "default": "OrganizationGroups",
+            "in": "query",
+            "type": "string",
+            "required": true,
+            "x-ms-summary": "$expand"
```

Comment:
@mint-virgil The 'description' property is required.

Reply:
updated in the latest commit